code/__DEFINES/~skyrat_defines/opposing_force_defines.dm

```diff
@@ -55,3 +55,4 @@
 #define OPFOR_EQUIPMENT_CATEGORY_SERVICES "Services"
 #define OPFOR_EQUIPMENT_CATEGORY_SPELLS "Spells"
 #define OPFOR_EQUIPMENT_CATEGORY_IMPLANTS "Implants"
+#define OPFOR_EQUIPMENT_CATEGORY_POINTLESS_BADASSERY "(Pointless) Badassery"
```

Comment:
This doesn't fit stylistically. Change it to just Pointless Badassery.

Reply:
This is the same name as how it's called on the uplink.

Reply:
Just because something is existing in code doesn't mean you can't improve it when the time comes up. This makes your menu item misaligned with all the other ones.